src/Cmts.mli

```diff
@@ -130,6 +130,9 @@ val remaining_comments : t -> (Cmt.t * string * Sexp.t) list
 
 val remaining_locs : t -> Location.t list
 
+val remaining_before : t -> Location.t -> Cmt.t list
+(** Returns comments before [loc] *)
```

Comment:
I would maybe change to something like: `[remaining_before c loc] returns comments before [loc].`

Reply:
Yes that's better.